app/views/items/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="item-content container">
+  <h1 class="my-4">
+    <%= @item.name %>
+  </h1>
+  <div class="row">
+    <div class="col-md-8">
+      <img class="img-responsive w-100" src="<%= @item.image %>" alt="">
+    </div>
+    <div class="col-md-4">
+      <h3 class="my-3">Item Description</h3>
+      <p><%= @item.description %></p>
+      <h3 class="my-3">Item Details</h3>
+      <ul class="list-unstyled">
+        <li>Merchant Name: <%= @item.user.name %></li>
+        <li>Available Quantity: <%= @item.inventory %></li>
+        <li>Price: $<%= @item.price %></li>
+        <li>Average Time to Ship: <%= @item.average_days_to_fulfill.round %> Days</li>
+      </ul>
+      <% if current_user.nil? || current_user.default? %>
+        <%= link_to "Add to Cart", root_path %>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
I'm wondering how this will look with the seed file I just pushed. We might need to refactor just a heads up because my descriptions are long. Or shorten them.

Reply:
it should wrap correctly.